content/rs/administering/designing-production/security/ldap-integration.md

```diff
@@ -44,7 +44,7 @@ the LDAP server.
 ### Step 1: Configure LDAP options in config file
 
 ```src
-vi /tmp/saslauthd.conf
+vi /opt/redislabs/config/saslauthd.conf
```

Comment:
This is wrong. The location _should_ be temporary.
The cluster writes the file provided to rladmin to this location on it's own (and the user has no need to know this)